source/docs/software/dashboards/dashboard-intro.rst

```diff
@@ -0,0 +1,40 @@
+
+Specific Dashboards (oldest to newest)
+--------------------------------------
+
+.. note:: SmartDashboard and Shuffleboard have a long history of aiding FRC teams. However, they do not have a person to maintain them so are not receiving bug fixes or improvements. Notably, Shuffleboard may experience performance issues on some machines under certain scenarios.  PRs from external contributors will be reviewed.
+
+:ref:`LabVIEW Dashboard <docs/software/dashboards/labview-dashboard/index:LabVIEW Dashboard>` (Driver / Programming) - easy to use and provides a lot of features straight out of the box like: camera streams, autonomous selection, and joystick feedback.  It can be customized using LabVIEW by creating a new Dashboard project.  While it :ref:`can be used <docs/software/dashboards/labview-dashboard/using-the-labview-dashboard-with-c++-java-code:Using the LabVIEW Dashboard with C++/Java Code>` by Java or C++ teams, they generally prefer SmartDashboard or Shuffleboard which can be customized in their respective language.
+
+:ref:`SmartDashboard <docs/software/dashboards/smartdashboard/index:SmartDashboard>` (Driver) - simple and efficient dashboard that uses relatively few computer resources.  It does not have the fancy look or some of the features Shuffleboard has, but it displays network tables data with a variety of widgets without bogging down the driver station computer.
+
+:ref:`Shuffleboard <docs/software/dashboards/shuffleboard/index:Shuffleboard>` (Driver) - straightforward and easily customizable dashboard. It displays network tables data using a variety of widgets that can be positioned and controlled with robot code. It includes many extra features like: tabs, recording / playback, and advanced custom widgets.
+
+:ref:`Glass <docs/software/dashboards/glass/index:Glass>` (Programming) - robot data visualization tool. Its GUI is extremely similar to that of the :ref:`Simulation GUI <docs/software/wpilib-tools/robot-simulation/simulation-gui:Simulation Specific User Interface Elements>`. In its current state, it is meant to be used as a programmer's tool rather than a proper dashboard in a competition environment.
+
+:ref:`AdvantageScope <docs/software/dashboards/glass/index:Glass>` (Programming) - robot diagnostics, log review/analysis, and data visualization application.  It reads logs in WPILOG, DS log, and RLOG file formats, plus live robot data viewing using NT4 or RLOG streaming.
```

Comment:
Correct.

This was shamelessly stolen from the [AdvantageScope page](https://github.com/Mechanical-Advantage/AdvantageScope/tree/fdda52f045c2953d5c3c9e3dd43b2b81ffdb634a)

I believe the file extensions are: .wpilog, .dslog, and .rlog

I don't know anything about rlog.  I guess the question is do we indicate the extension as all caps as shown or `.wpilog`.  I have seen it both ways.

@jwbonner

Reply:
I think most places in the docs currently refer to it as a “WPILib Data Log” instead of using an abbreviation. I’ve also seen it as WPILog. If we standardize on capitalization/formatting I’m happy to update the AdvantageScope docs too for consistency.

RLOG appears in all caps everywhere in our docs. But honestly I think that should just be removed here - it’s a legacy format used by AdvantageKit in 2022, and we’ve kept it for backwards compatibility with old logs. New users will never have a use for it.

Reply:
This would be my preference.  Objections?

"It reads the WPILib Data Log (``.wpilog``) and Driver Station Log (``.dslog``) file formats, plus live robot data viewing using NT4."

Reply:
I would just amend it to something like "Driver Station Log (`.dslog`/`.dsevents`)" since the DS console log is supported too. Otherwise it looks good to me.